src/releases/views/ServiceDetailView.js

```diff
@@ -0,0 +1,237 @@
+import React, {useState, useRef} from 'react';
+import {Helmet} from 'react-helmet';
+import {useQuery, useMutation} from '@apollo/react-hooks';
+import {withRouter} from 'react-router-dom';
+import {TaskList} from '../components/TaskList';
+import {Events} from '../components/Events';
+import {
+  Segment,
+  Container,
+  Button,
+  Dimmer,
+  Divider,
+  Grid,
+  Header,
+  Form,
+  Icon,
+  Image,
+  Label,
+  Loader,
+  Message,
+  Modal,
+} from 'semantic-ui-react';
+import paragraph from '../../assets/paragraph.png';
+import UpdateServiceForm from '../forms/UpdateServiceForm';
+
+import {UPDATE_SERVICE} from '../mutations';
+import {GET_SERVICE, ALL_EVENTS} from '../queries';
+
+const ServiceDetailView = ({match}) => {
+  const [editing, setEditing] = useState(false);
+
+  const relayId = Buffer.from(
+    'TaskServiceNode:' + match.params.serviceId,
+  ).toString('base64');
+
+  const {
+    loading: serviceLoading,
+    error: serviceError,
+    data: serviceData,
+  } = useQuery(GET_SERVICE, {
+    variables: {id: relayId},
+    context: {clientName: 'coordinator'},
+  });
+
+  const {
+    loading: eventsLoading,
+    error: eventsError,
+    data: eventsData,
+  } = useQuery(ALL_EVENTS, {
+    variables: {taskService: relayId, first: 10},
+    context: {clientName: 'coordinator'},
+  });
+
+  const service = serviceData && serviceData.taskService;
+
+  const [
+    updateService,
+    {loading: updateServiceLoading, error: updateServiceError},
+  ] = useMutation(UPDATE_SERVICE, {
+    context: {clientName: 'coordinator'},
+  });
+
+  const formRef = useRef();
+
+  const toggle = ev => {
+    updateService({
+      variables: {
+        taskService: service.id,
+        input: {
+          name: service.name,
+          url: service.url,
+          enabled: !service.enabled,
+        },
+      },
+    });
+  };
+
+  if (serviceError)
+    return (
+      <Container as={Segment} basic>
+        <Message
+          negative
+          header="Error"
+          content={serviceError.message}
+        />
+      </Container>
+    );
+
+  if (service || serviceLoading) {
```

Comment:
I think you want `!service || serviceLoading` here or else it will always be loading.